supervisor/api/docker.py

```diff
@@ -30,10 +31,43 @@
     }
 )
 
+# pylint: disable=no-value-for-parameter
+SCHEMA_OPTIONS = vol.Schema({vol.Optional(ATTR_ENABLE_IPV6): vol.Boolean()})
+
 
 class APIDocker(CoreSysAttributes):
     """Handle RESTful API for Docker configuration."""
 
+    @api_process
+    async def info(self, request: web.Request):
+        """Get docker info."""
+        data_registries = {}
+        for hostname, registry in self.sys_docker.config.registries.items():
+            data_registries[hostname] = {
+                ATTR_USERNAME: registry[ATTR_USERNAME],
+            }
+        return {
+            ATTR_VERSION: self.sys_docker.info.version,
+            ATTR_ENABLE_IPV6: self.sys_config.enable_ipv6,
+            ATTR_STORAGE: self.sys_docker.info.storage,
+            ATTR_LOGGING: self.sys_docker.info.logging,
+            ATTR_REGISTRIES: data_registries,
+        }
+
+    @api_process
+    async def options(self, request: web.Request) -> None:
+        """Set docker options."""
+        body = await api_validate(SCHEMA_OPTIONS, request)
+
+        if ATTR_ENABLE_IPV6 in body:
+            self.sys_config.enable_ipv6 = body[ATTR_ENABLE_IPV6]
+
+        # Save changes before processing addons in case of errors
+        await self.sys_updater.save_data()
+        await self.sys_config.save_data()
+
+        await self.sys_resolution.evaluate.evaluate_system()
```

Comment:
We don't need all the logic from the Supervisor option call, as we don't change properties which might be relevant for updater/resolution system.

So this should boil down to:

```suggestion
            self.sys_docker.config.enable_ipv6 = body[ATTR_ENABLE_IPV6]

        await self.sys_docker.config.save_data()
```